### ISSUE TITLE
Design Opengraph/Social/Twitter card image

### ISSUE THREAD

**Issue by rugk** [rugk/mastodon-simplified-federation#29]
To be used for GitHub and AMO. (shown when you share a link to them on Twitter or so)

Similar to the ones from my other add-ons:
* https://github.com/rugk/awesome-emoji-picker/blob/master/assets/repo-card-opengraph.png
* https://github.com/rugk/offline-qr-code/blob/master/assets/repo-card-opengraph.png

## Requirements

* to be saved and design via SVG (e.g. Inkscape). Use this template (template based on official GitHub template.) https://github.com/rugk/opengraph-template/
* show logo and/or screenshots (if it fits), some kind of indication that it is an Firefox add-on (i.e. Firefox logo) and some kind of indicator that it is on GitHub/FLOSS (GitHub logo)
* few text, big text, great background
* looks good! :blush:

**Comment by bull500** [comment]
Hello @rugk i'd like to take up this issue. Have made a sample
![mastaddonban](https://user-images.githubusercontent.com/8917275/92093294-ce761300-edf0-11ea-96ab-07e7ac750fbd.png)

**Comment by SuperSandro2000** [comment]
@bull500 The Firefox logo is quite big compared to the mastodon logo. Mastodon also works in all other major browers and the floss thing looks totally out of place.

**Comment by rugk** [comment]
So first of all thanks for creating this and trying something.

Some feedback I'd give:
* The GitHub badge should be inverted, i.e. white, so it can be read on the dark background.
* I'd remove the FLOSS thing. The GitHub icon is enough to show it's FLOSS.
* there seem to be a lot of fonts in the image, which is quite confusing. Maybe we can try to reduce those? (Would be great to get the SVG source file, so I can have a look at which fonts you've used.)
* I'd place the name of the extension, i.e. "mastodon simplified federation" on one line, if possible
* also, somewhere the extension logo should be placed hmm… I know difficult if we also want to display the original logo

Apart from this I like this idea.

@SuperSandro2000 Yes, Mastodon works in other browsers, but this add-on (currently¹) does not.

¹ please create an issue if you want this to be changed.

**Comment by SuperSandro2000** [comment]
Oh nvm. I thought I was on the main mastodon repo. Sorry about that.

**Comment by bull500** [comment]
I've redone the image with the changes requested
i had only used a single font(Free Sans) in different variations.
The other two come from the logo's of the respective softwares. 
![rect502](https://user-images.githubusercontent.com/8917275/92149247-67ca1700-ee3b-11ea-8c1f-1297ae8cdc85.png)
[mastofinal-template.svg.2020_09_03_22_44_33.0.zip](https://github.com/rugk/mastodon-simplified-federation/files/5170301/mastofinal-template.svg.2020_09_03_22_44_33.0.zip)

**Comment by rugk** [comment]
Okay, great! 
The only thing that is problematic now, but I don't know how to solve is that we have two very similar icons now that may be confusing:
* the mastodon icon
* _and_ the add-on icon

Some ideas:
* make the mastodon icon a small subscript (i.e. ¹) So Mastodon¹, where "¹" is the icon? Just like [the Fedora icon does it](https://commons.wikimedia.org/wiki/File:Fedora_logo_and_wordmark.svg)?
* show them stacked somehow, so you don't see the whole icon?
* somehow indicate that the mastodon icon" changes/moves/… into the mastodon icon?
* any other ideas??

**Comment by bull500** [comment]
Thank you for response @rugk 

im not sure if modifying logo's of those OSS projects would create problems.
The other option is to use the text+logo of both github and mastodon.
Masto's types are available here - https://joinmastodon.org/press-kit.zip
GitHub we mostly have to combine the text and the logo

![rect503](https://user-images.githubusercontent.com/8917275/92529183-b155a000-f247-11ea-9127-c354e9dbbf10.png)

**Comment by bull500** [comment]
i tried a bunch with the arrow in the back ground.
Hard to get it right for now.
Increasing icon kinda loses the balance of the title next to it

Type 1:
![path2-36](https://user-images.githubusercontent.com/8917275/93187246-3b789980-f75d-11ea-95fc-49579617efb2.png)

Type 2:
![g3808](https://user-images.githubusercontent.com/8917275/93187314-4fbc9680-f75d-11ea-8797-2d8507cffd01.png)

**Comment by paulo-roger** [comment]
I loved the type 1, the other reminds me of amazon ):

**Comment by rugk** [comment]
Indeed, that's also my favourite. @bull500 do you still have source files (svg) of that?

**Comment by rugk** [comment]
@bull500 would really love to get the SVG files for that, if you still have them.
